int/tests/test_presentation_integration.py

```diff
@@ -0,0 +1,77 @@
+from acapy_client.models.indy_proof_request import IndyProofRequest
```

Comment:
I think I'd rename this file to `test_presentation` since it's already in the integration test folder.

Reply:
We now have a duplicate file :grimacing: